devicetypes/smartthings/fortrezz-water-valve.src/fortrezz-water-valve.groovy

```diff
@@ -19,8 +19,8 @@ metadata {
 		capability "Refresh"
 		capability "Sensor"
 
-		fingerprint deviceId: "0x1000", inClusters: "0x25,0x72,0x86,0x71,0x22,0x70"
-		fingerprint mfr:"0084", prod:"0213", model:"0215", deviceJoinName: "FortrezZ Water Valve"
+		fingerprint deviceId: "0x1000", inClusters: "0x25,0x72,0x86,0x71,0x22,0x70", deviceJoinName: "FortrezZ Valve"
```

Comment:
I think the manufacturer name should be left off here since it's a generic fingerprint.

Reply:
But the manufacturer name is still in the name of DTH, so it won't change anything in that area.

Reply:
OK, we can leave it. It does feel like something we can improve later since generic fingerprints could match any manufacturer's device.

Reply:
If the code should work with any device that matches that fingerprint, the `deviceJoinName` could be set to something generic, like just "Water Valve".

Reply:
I agree @dkirker but in the interest of getting this moving I think it's OK since the current behavior is to use the DTH name which had the manufacturer name in there already.